doc/sphinx/language/core/definitions.rst

```diff
@@ -13,15 +13,18 @@ Let-in definitions
 .. prodn::
    term_let ::= let @name {? : @type } := @term in @term
    | let @name {+ @binder } {? : @type } := @term in @term
-   | let ( {*, @name } ) {? {? as @name } return @term100 } := @term in @term
-   | let ' @pattern := @term {? return @term100 } in @term
-   | let ' @pattern in @pattern := @term return @term100 in @term
-
-:n:`let @ident := @term in @term’`
-denotes the local binding of :n:`@term` to the variable
-:n:`@ident` in :n:`@term`’. There is a syntactic sugar for let-in
-definition of functions: :n:`let @ident {+ @binder} := @term in @term’`
-stands for :n:`let @ident := fun {+ @binder} => @term in @term’`.
+   | @destructuring_let
+
+:n:`let @ident := @term__1 in @term__2`
+denotes the local binding of :n:`@term__1` to the variable
+:n:`@ident` in :n:`@term__2`. There is a syntactic sugar for let-in
```

Comment:
```suggestion
denotes the local binding of the variable :n:`ident` to the value :n:`@term__1`
in :n:`@term__2`. There is a syntactic sugar for let-in
```

One might also say "represents" rather than "denotes"--a much more common word.  Everyone knows the former, many people don't know the latter.  Among college grads I think many (most?) would know "denotes".